test/integration/rules/form-field-label/form-field-label.html

```diff
@@ -0,0 +1,40 @@
+<!-- Inapplicable -->
+<input id='inapplicable1' role="presentation" />
```

Comment:
1. remove role, they should be inapplicable irrespective

2. add textarea tests